web/client/components/catalog/RecordItem.css

```diff
@@ -27,6 +27,11 @@
     overflow: hidden;
 }
 
+.record-item-description {
+    width: 350px;
```

Comment:
If possible we prefer to avoid fixed width, in particular for nested elements.
This because children elements should adapt their size based on parent element, in this case the catolog panel.
Could you verify if it still works removing width in particular with description without white spaces?

Reply:
It didn't work when I removed width, therefore have done another implementation which uses lodash to achieve the same results .